### PR TITLE
Feature/44 lobby scene

### LINKED ISSUE #44: LobbyScene
**Description**
Create LobbyScene. Make players join a slot on button click. Ready functionality. Once all are ready start countdown (5s) and start game/swap scene

**Subtasks**
- [ ] Joining on button click
- [ ] Ready functionality
- [ ] Countdown ->  StartGame

### PR DESCRIPTION
**Pull request by tzaarela**
Closes #44 

**Description**  
Now able to join the lobby with 4 players

**List of changes**  
- added lobby.cs
- added Start as an inputAction to "Gamepad Start and keyboard Enter"

### CODE REVIEW

File: Assets/00_Content/Scripts/Menu/Lobby.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+using Player;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Menu {
+
+    public class Lobby : MonoBehaviour
```
Comment (MrJackey) [inline Assets/00_Content/Scripts/Menu/Lobby.cs:10]: Would it be fitting to place the lobby script on the canvas or an object in the canvas?

File: Assets/00_Content/Scripts/Menu/PlayerSlotObject.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        private bool isTaken;
+        public bool IsTaken => isTaken;
+
+        private PlayerComponent playerComponent;
+        public PlayerComponent PlayerComponent => playerComponent;
+
+        [SerializeField] TextMeshProUGUI joinText;
+        [SerializeField] TextMeshProUGUI readyText;
+
+        private Image background;
```
Comment (MrJackey) [inline Assets/00_Content/Scripts/Menu/PlayerSlotObject.cs:24]: The order of these fields is a bit confusing, perhaps a reordering would improve clarity

**Review by MrJackey — CHANGES_REQUESTED** [review]
Canvas scaling probably shouldn't be constant pixel size, also I can't seem to start the game. Otherwise it seems to work well, just had a few thoughts

**Review by smeas — APPROVED** [review]
👍

**Pull request merged**